package.json

```diff
@@ -27,7 +27,7 @@
     "css-loader": "0.28.8",
     "extract-text-webpack-plugin": "2.1.2",
     "js-cookie": "2.2.0",
-    "node-sass": "4.7.2",
+    "node-sass": "4.14",
```

Comment:
I think that we need to add the change to the package-lock.json file after this change

Reply:
Done